e2e_playwright/shared/dataframe_utils.py

```diff
@@ -88,6 +86,56 @@ def calc_middle_cell_position(
     return column_middle_width_px, row_middle_height_px
 
 
+def unfocus_dataframe(page: Page) -> None:
```

Comment:
I am wondering whether we can give this a generic name as the unfocus should probably work for all elements (e.g. text input etc.)

Reply:
This method is slightly more specific to dataframe since we also press `escape` to remove the current cell selection. `escape` might have unintended effects for other elements or might be necessary. But it would probably be good to also have a more generic version in `app_utils` that can be used for most elements.